packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -376,34 +413,39 @@ class FlutterErrorBuilder {
       level: level,
     ));
   }
-
+  
+  // TODO(inmatrix): ask what this method is for.
   void addBlock(String name, FlutterErrorBuilder blockContents) {
```

Comment:
@jacob314 Do you intend to keep this method? What is it for?

Reply:
Removed it. It wasn't needed in the end with the helpers added for stack traces and enumerating lists of elements.

Reply:
Okay, I'll remove it from my PR.